packages/storybook/stories/Stateful Client/SettingUpReactHooks.stories.mdx

```diff
@@ -60,4 +60,4 @@ ReactDOM.render(
 
 Once you have wrapped your entire app with providers, you are now can use the usePropsFor() method inside of your app to connect your components to the state.
 
-To see a full end-to-end example using providers, see [Getting Started with StatefulClient](./?path=/docs/quickstarts-statefulclient--page)
+To see a full end-to-end examples using providers, see [Getting Started with Call Stateful Client](./?path=/docs/quickstarts-statefulcallclient--page) and [Getting Started with Chat Stateful Client](./?path=/docs/quickstarts-statefulchatclient--page).
```

Comment:
examples should be example,
```suggestion
To see a full end-to-end example using providers, see [Getting Started with Call Stateful Client](./?path=/docs/quickstarts-statefulcallclient--page) and [Getting Started with Chat Stateful Client](./?path=/docs/quickstarts-statefulchatclient--page).
```
Match singularity of 'a'

Reply:
Removed `a`, thanks!